lib/embulk/input/hubspot/deal.rb

```diff
@@ -51,13 +47,18 @@ def guess_deal(api_key)
               config.api_key["hapikey"] = api_key
             end
             basic_api = Hubspot::Crm::Deals::BasicApi.new
-            deal_data = basic_api.get_page(auth_names: "hapikey").results
+            deal_data = basic_api.get_page(properties: get_all_deal_property, auth_names: "hapikey").results
             deal_data.map do |deal|
               { "id" => deal.id, "createdAt" => deal.created_at.to_time, "updatedAt" => deal.updated_at.to_time, "archived" => deal.archived }.merge(deal.properties)
             end
           end
 
           private
+          def get_all_deal_property
+            property_api = Hubspot::Crm::Properties::CoreApi.new
+            all_deals_property = property_api.get_all("deal", auth_names: "hapikey")
+            all_deals_property.results.map { |property| property.name}
```

Comment:
nits rubocopとかのlinterを走らせておいたら良さそうです。

```suggestion
            all_deals_property.results.map { |property| property.name }
```